src/strategies/types/types.ts

```diff
@@ -120,6 +121,16 @@ export namespace VerifyFunction {
       req?: Request,
     ): Promise<T | null>;
   }
+  export interface Auth0Fn<T = IAuthUser> extends GenericAuthFn<T> {
```

Comment:
Why this ?

Reply:
We create interface for each strategy based on its verify function type